darwin/app-config.zsh

```diff
@@ -32,7 +32,7 @@ where git &> /dev/null
 
 
 # swmm target created by the cmake build script
-TEST_CMD="run${PROJECT}"
+TEST_CMD="run-${PROJECT}"
```

Comment:
One of the features of CI tools is support for SWMM and EPANET projects. So we either need to establish a uniform name for the executable or have a logical statement based on the PROJECT variable to handle project specific naming convention.